com.adsbynimbus.nimbus/Editor/NimbusManagerCreator.cs

```diff
@@ -184,7 +184,7 @@ public class NimbusManagerCreator : EditorWindow {
 				#endif
 
 				AssetDatabase.CreateAsset(_asset,
-					"Packages/com.adsbynimbus.unity/Runtime/Scripts/Nimbus.ScriptableObjects/NimbusSDKConfiguration.asset");
```

Comment:
any reason why the package name is being changed @JasonSznol . That path is tied to the Unity package delivery system.